bot/seasons/evergreen/recommend_game.py

```diff
@@ -0,0 +1,51 @@
+import json
+import logging
+from pathlib import Path
+from random import shuffle
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+game_recs = []
+
+# Populate the list `game_recs` with resource files
+for rec_path in Path("bot/resources/evergreen/game_recs").glob("*.json"):
+    with rec_path.open(encoding='utf-8') as file:
+        data = json.load(file)
+    game_recs.append(data)
+shuffle(game_recs)
+
+
+class RecommendGame(commands.Cog):
+    """Commands related to recommending games."""
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.index = 0
+
+    @commands.command(name="recommendgame", aliases=['gamerec'])
+    async def recommend_game(self, ctx):
```

Comment:
Similarly, missing function annotations for the parameter and return value.
```suggestion
    async def recommend_game(self, ctx: commands.Context) -> None:
```